Docs/Configuration.tex

```diff
@@ -4646,6 +4646,21 @@ \subsection{Entry Properties}\label{miscentryprops}
   \textbf{Description}: Arbitrary ASCII string used to provide a human readable
   reference for the entry. Whether this value is used is implementation defined.
 
+\item
+  \texttt{DisableFirmwareRuntime}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Description}: Disable firmware runtime during usage of a tool.
+
+  This disables all of the firmware runtime protections provided by
+  \texttt{OpenRuntime.efi} during the time a tool is in use. It should
+  normally be avoided, but may be required for instance if a tool needs
+  to access NVRAM directly without the redirects put in place by
+  \texttt{RequestBootVarRouting}.
+
+  \emph{Note}: This property is only valid for \texttt{Tools} and cannot be
```

Comment:
We usually use `option` instead of `property` elsewhere in our docs; better change it for consistency :)